supabase/migrations/20240528091033_add_completed_parcels_view.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE VIEW completed_parcels AS
```

Comment:
This sql is a little bit gnarly and might need some explanation - any suggestions as to what to do here, or is it fine?